_locales/en/messages.json

```diff
@@ -2480,6 +2480,45 @@
     "configurationMagHardware": {
         "message": "Magnetometer (if supported)"
     },
+    // ---------------------------------------------
```

Comment:
This is not valid JSON. Please remove. Also, there are a lot of other messages in here that will have to be removed when the functionality to support older versions gets removed, so marking these specifically does not buy us anything.